Gemfile.lock

```diff
@@ -0,0 +1,13 @@
+GEM
```

Comment:
It's auto generated, but it's meant to track dependency versions so is usually checked into version control (apparently).